pkg/controller/upload_controller.go

```diff
@@ -103,17 +104,30 @@ func (c *uploadController) enqueueUploadItem(obj interface{}) {
 	switch req.Status.Phase {
 	case "", pluginv1api.UploadPhaseNew, pluginv1api.UploadPhaseInProgress, pluginv1api.UploadPhaseUploadError:
 		// Process New and InProgress and UploadError Uploads
+	case pluginv1api.UploadPhaseCanceled:
```

Comment:
Move this to the top. Do we want the UploadPhaseNew and others to fall through to the default case?

Reply:
golang doesn't have fallthrough for switch statement.